contracts/test/TestCAssetProxy.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.5.16;
+
+import "../WrappedPosition.sol";
+import "./TestCToken.sol";
+
+contract TestCAssetProxy is WrappedPosition {
```

Comment:
instead of copy-pasting the whole code can't we inherit the main code and make the function virtual to override as we want. It will be more cleaner approach